### ISSUE TITLE
Tizonia stops responding to keyboard after bg/fg switch

### ISSUE THREAD

**Issue by Zamlet** [tizonia/tizonia-openmax-il#725]
# Issue Template

### Description
After launching Tizonia, sending it to the background then bringing it back to the foreground causes it to stop responding to keyboard navigation ([p] previous, [n] next, [m] mute etc.).

### Steps to Reproduce
1. Launch Tizonia in console window and begin playing music.
2. Test keyboard controls.  They work as expected.
3. Send Tizonia to background by pressing Ctrl-Z.  Music stops and prompt appears.
4. Type _fg_ at prompt and press Enter.  Music resumes.
5. Test keyboard controls.  They no longer work.

**Expected behavior:** [What you expect to happen]
After resuming play by bringing Tizonia to foreground again, keyboard controls should work.

**Actual behaviour:** [What actually happens]
After resuming play by bringing Tizonia to foreground again, keyboard controls do not work.

**Reproduces how often:** [What percentage of the time does it reproduce?]
100% reproducible for me so far.

### Versions
From `tizonia --debug`:
----------
tizonia 0.21.0. Copyright (C) 2020 Juan A. Rubio and contributors
This software is part of the Tizonia project <https://tizonia.org>

Debug Info:
	    * [Linux@5.3.0-46-generic-#38~18.04.1-Ubuntu SMP Tue Mar 31 04:17:56 UTC 2020]
	    * [Boost 1_65_1]
	    * [TagLib 1.11.1]
	    * [MediaInfoLib - v17.12]
----------

From `cat /etc/os-release`:
----------
NAME="Linux Mint"
VERSION="19.3 (Tricia)"
ID=linuxmint
ID_LIKE=ubuntu
PRETTY_NAME="Linux Mint 19.3"
VERSION_ID="19.3"
HOME_URL="https://www.linuxmint.com/"
SUPPORT_URL="https://forums.ubuntu.com/"
BUG_REPORT_URL="http://linuxmint-troubleshooting-guide.readthedocs.io/en/latest/"
PRIVACY_POLICY_URL="https://www.linuxmint.com/"
VERSION_CODENAME=tricia
UBUNTU_CODENAME=bionic
----------

### Installation Method Used
Installed originally (v0.16??) from Ubuntu/Mint repositories, and upgraded by same method.

### Additional Information
Can't think of anything to add here, sorry.  Above report pretty much covers it.

**Comment by juanrubio** [comment]
Hi @Zamlet !

Thanks for reporting this.

It is true that when the application is back in the foreground, the keyboard shortcuts don't seem to work. However, if you press 'Enter' key, things go back to normal.

e.g
```
$ tizonia --gmusic-library
Ctrl+z 
suspended (signal) tizonia --gmusic-library
$ fg
continued tizonia --gmusic-library
[Enter]
keyboard shortcuts working now
```

**Comment by Zamlet** [comment]
Confirmed - you are right.  But I don't understand why this is the case, nor am I clear whether or not tizonia is causing this behavior or could prevent it somehow.

**Comment by juanrubio** [comment]
Yes, it is very possible that this could be improved. And it might probably be an easy one.

Please feel free to submit a PR. That would be most welcome.

**Comment by thepyguy** [comment]
I would kind of expect to continue playing audio while in the background. Not really a big deal...I just leave a terminal open in a different workspace for tizonia.

**Comment by juanrubio** [comment]
@thepyguy 

> Send Tizonia to background by pressing Ctrl-Z

Note that using Ctrl-Z does not send it to the background, it only suspends it. So audio output is not expected. 

I.e. after Ctrl-Z You need to type 'bg' to resume the application in the background.

**Comment by thepyguy** [comment]
Ahh I see makes sense. I've had a misconception for a long time lol.
